<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WCamera</class>
 <widget class="QWidget" name="WCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="anticameraBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Anticamera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="balconeBtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Balcone</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ArmadioCassettieraBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Armadio e Cassettiera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ScrivaniaComodinoBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Scrivania e Comodino</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
